KuduSync.NET/KuduSync.cs

```diff
@@ -387,7 +399,8 @@ private DirectoryInfoBase CreateDirectoryInfo(string path)
 
         private bool IgnorePath(FileSystemInfoBase fileSystemInfoBase)
         {
-            return _ignoreList.Contains(fileSystemInfoBase.Name);
+            return (_ignoreList.Contains(fileSystemInfoBase.Name)
+                || _wildcardIgnoreList.Any((name) => fileSystemInfoBase.Name.EndsWith(name, StringComparison.OrdinalIgnoreCase)));
```

Comment:
correct. 
and we put "_ignoreList.Contains(fileSystemInfoBase.Name)" first, if we find entry from _ignoreList list. the later condition shouldn`t be executed.